sykefi/marinedataportal
# fix: #14630 Fix depth range and focus

Testing:
- Select water quality and some parameter
- Select depth range
- Go through the form and check that the query works as expected
